test/downloadutils.mjs

```diff
@@ -25,89 +24,74 @@ function rewriteWebArchiveUrl(url) {
   // Web Archive URLs need to be transformed to add `if_` after the ID.
   // Without this, an HTML page containing an iframe with the PDF file
   // will be served instead (issue 8920).
-  var webArchiveRegex =
+  const webArchiveRegex =
     /(^https?:\/\/web\.archive\.org\/web\/)(\d+)(\/https?:\/\/.+)/g;
-  var urlParts = webArchiveRegex.exec(url);
+  const urlParts = webArchiveRegex.exec(url);
   if (urlParts) {
-    return urlParts[1] + (urlParts[2] + "if_") + urlParts[3];
+    return `${urlParts[1]}${urlParts[2]}if_${urlParts[3]}`;
   }
   return url;
 }
 
-function downloadFile(file, url, callback, redirects) {
+function downloadFile(file, url, redirects) {
```

Comment:
Could we use a default parameters here instead, since that's (another) thing that didn't exist back when this code was written?
```suggestion
function downloadFile(file, url, redirects = 0) {
```

Reply:
Yes, this combined with the second suggestion below looks nicer; thanks!